demo/admin/src/products/ManufacturerForm.tsx

```diff
@@ -82,46 +82,43 @@ export function ManufacturerForm({ id }: FormProps): React.ReactElement {
         id ? { variables: { id } } : { skip: true },
     );
 
+    const filteredData = data ? filter<GQLManufacturerFormDetailsFragment>(manufacturerFormFragment, data.manufacturer) : undefined;
```

Comment:
IMO it would be better if this would be inside the `useMemo` call.

Reply:
@johnnyomair good idea, changed.